src/frontends/pytorch/src/op/rand.cpp

```diff
@@ -32,40 +34,14 @@ OutputVector make_random_normal(const NodeContext& context,
                                 const Output<Node>& mean_const) {
     std::random_device rd;
     std::mt19937 gen(rd());
-    std::uniform_int_distribution<uint64_t> distrib(0, 9999);
+    std::uniform_real_distribution<float> distrib(0.0f, 9999.0f);
+    float seed = distrib(gen);
 
-    const uint64_t global_seed = 0;
-
-    const uint64_t seed_1 = distrib(gen);
-    const uint64_t seed_2 = distrib(gen);
-
-    auto min_val = context.mark_node(v0::Constant::create(target_type, Shape{1}, {std::numeric_limits<float>::min()}));
-    auto max_val = context.mark_node(v0::Constant::create(target_type, Shape{1}, {1}));
-
-    auto uniform_1 = context.mark_node(
-        std::make_shared<v8::RandomUniform>(sizes, min_val, max_val, target_type, global_seed, seed_1));
-    auto uniform_2 = context.mark_node(
-        std::make_shared<v8::RandomUniform>(sizes, min_val, max_val, target_type, global_seed, seed_2));
-
-    // Compute Box–Muller transform
-    // random_normal = scale * ng.sqrt(-2.0 * ng.log(uniform_1)) * ng.cos(2.0 * np.pi * uniform_2) + mean
-    auto pi = context.mark_node(v0::Constant::create(target_type, Shape{1}, {3.141592653589793}));
-    auto minus_two = context.mark_node(v0::Constant::create(target_type, Shape{1}, {-2.0}));
-    auto two = context.mark_node(v0::Constant::create(target_type, Shape{1}, {2.0}));
-
-    auto log = context.mark_node(std::make_shared<v0::Log>(uniform_1));
-    auto multiply_minus_two_log = context.mark_node(std::make_shared<v1::Multiply>(log, minus_two));
-    auto sqrt = context.mark_node(std::make_shared<v0::Sqrt>(multiply_minus_two_log));
-
-    auto multiply_two_pi = context.mark_node(std::make_shared<v1::Multiply>(uniform_2, pi));
-    auto multiply_two_pi_uniform_2 = context.mark_node(std::make_shared<v1::Multiply>(multiply_two_pi, uniform_2));
-    auto cos = context.mark_node(std::make_shared<v0::Cos>(multiply_two_pi_uniform_2));
-
-    auto sqrt_x_cos = context.mark_node(std::make_shared<v1::Multiply>(sqrt, cos));
-    auto product = context.mark_node(std::make_shared<v1::Multiply>(scale_const, sqrt_x_cos));
-    auto sum = context.mark_node(std::make_shared<v1::Add>(product, mean_const));
-
-    return {sum};
+    auto res_pair = ov::frontend::make_random_normal(sizes, target_type, mean_const, scale_const, seed);
+    for (const auto& node : res_pair.second) {
+        context.mark_node(node);
```

Comment:
There is `mark_nodes` function for such cases.

Reply:
ready